src/Common/MemoryTracker.h

```diff
@@ -154,14 +157,6 @@ class MemoryTracker
     {
         return soft_limit.load(std::memory_order_relaxed);
     }
-    void setAllowUseJemallocMemory(bool value)
```

Comment:
I guess `allow_use_jemalloc_memory` can be marked as deprecated now in server settings? And also removed from the `test_global_overcommit_tracker` test

Reply:
So as `cgroup_memory_watcher_hard_limit_ratio` and `cgroup_memory_watcher_soft_limit_ratio`